apps/projects/app/components/Content/IssueDetail/DetailsCard.js

```diff
@@ -15,147 +16,109 @@ import { issueShape } from '../../../utils/shapes.js'
 import { IssueTitleLink } from '../../Panel/PanelComponents'
 import { formatDistance } from 'date-fns'
 
-const DeadlineDistance = ({ date }) =>
+const OpenedAgo = ({ date }) =>
   formatDistance(new Date(date), new Date(), { addSuffix: true })
 
-const SummaryCell = ({ label, children, grid }) => (
-  <div css={`
-    display: flex;
-    flex-direction: column;
-    grid-area: ${grid};
-    overflow: hidden;
-  `}>
-    <Label text={label} />
+const dot = <span css="margin: 0px 10px">&middot;</span>
+
+const Title = ({ issue }) => {
+  const theme = useTheme()
+
+  return (
     <div css={`
-      height: 100%;
       display: flex;
-      align-items: center;
-      margin-top: ${2 * GU}px;
+      flex-direction: column;
+      flex-basis: 100%;
+      flex: 2;
+      margin-right: 20px;
     `}>
-      {children}
+      <Text.Block size="xxlarge" style={{ marginBottom: 1.5 * GU + 'px' }}>
+        {issue.title}
+      </Text.Block>
+      <IssueTitleLink issue={issue}>
```

Comment:
Maybe this instead?

```js
<IssueTitleLink
  issue={issue}
  append={
    <>
      <Dot />
      <IconInfo ... />
      Opened ...
    </>
  }

Reply:
Or maaaybe we don't need to pass any new props to `IssueTitleLink`?

```js
<div css="display: flex">
  <IssueTitleLink issue={issue} />
  <Dot />
  <IconInfo ... />
  <span>Opened ...</span>
</div>

Reply:
With careful use of display: flex (actually checking all css directives) it was possible to achieve the same effect without passing any additional props/children to IssueTitleLink - like in the second snippet :)